.github/workflows/check-partner-approved-label.yml

```diff
@@ -0,0 +1,37 @@
+name: Partner-Approved label verification
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
+
+permissions:
+  contents: read
+
+jobs:
+  check_label_job:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/content' && github.event.pull_request.head.repo.fork == true && contains(github.head_ref, 'xsoar-bot-contrib-ContributionTestPack') == false && contains(github.event.pull_request.title, '[Marketplace Contribution]') == false
```

Comment:
Why do we need this condition? 
`contains(github.event.pull_request.title, '[Marketplace Contribution]') == false`
We also want it to run on PRs contributed through the marketplace - am I wrong?

Reply:
wrong condition, changing